test/unittest/ipam_bench/ipam_test.go

```diff
@@ -0,0 +1,362 @@
+package ipam_bench
+
+import (
+	"flag"
+	"fmt"
+	"k8s.io/utils/strings/slices"
+	"math/big"
+	"testing"
+
+	"crypto/rand"
+	"k8s.io/klog/v2"
```

Comment:
```suggestion
	"crypto/rand"
	"flag"
	"fmt"
	"math/big"
	"testing"

	"k8s.io/klog/v2"
	"k8s.io/utils/strings/slices"
```

Reply:
Why use `crypto/rand` instead of `math/rand`?

Reply:
1. What about the performance difference? AFAIK, `crypto/rand` is used mainly for security.
2. Before using `math/rand`, you need to initialize the random source:

```go
func init() {
	rand.Seed(time.Now().UnixNano())
}
```

Reply:
1.  no significant difference and I see crypto/rand is used in pkg/util/net.go  such as GenerateMac
2. got